school/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] The purpose of the `add_rental` method is to allow the user to create a rental given a person and a date, so this method should be more about the instantiation of the rental class.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```